Resources/Prototypes/Entities/Structures/Piping/Disposal/units.yml

```
@@ -120,6 +120,7 @@
     whitelist:
       components:
       - Item
+      - Mailable #Goobstation - Syndicate main uniform
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)